app/views/browse/service.html

```diff
@@ -64,23 +64,29 @@
                       <dd>{{service.spec.clusterIP}}</dd>
                       <dt>Session affinity:</dt>
                       <dd>{{service.spec.sessionAffinity}}</dd>
-                      <dt ng-if="resource.status.loadBalancer.ingress.length">Ingress points</dt>
-                      <dd ng-if="resource.status.loadBalancer.ingress.length">
-                        <span ng-repeat="ingress in resource.status.loadBalancer.ingress"
+                      <dt ng-if="service.status.loadBalancer.ingress.length">Ingress Points:</dt>
+                      <dd ng-if="service.status.loadBalancer.ingress.length">
+                        <span ng-repeat="ingress in service.status.loadBalancer.ingress"
                           >{{ingress.ip}}<span ng-if="!$last">, </span></span>
                       </dd>
-                      <dt>Routes:</dt>
-                      <dd>
+                      <dt ng-if="service.spec.externalIPs.length">External IPs:</dt>
```

Comment:
You should use ng-if-start and ng-if-end when you have consecutive sibling nodes checking the same thing.  We have some old code that didn't do this but any new code should.  Fix up the other instances in here like ingress above and routes below

ng-if-start="service.spec.externalIPs.length"